source/clear-linux/concepts/swupd-about.rst

```diff
@@ -1,80 +1,119 @@
 .. _swupd-about:
 
-Software update
-###############
+swupd (Software Updater)
+########################
 
-swupd is an OS-level software update program that applies updates to a
-|CL-ATTR| or derivative.
-
-While traditional Linux\*-based distributions rely on packages for software
-deployment, |CL-ATTR| uses bundles. In a traditional Linux\* distro,
-packages provide a particular utility or library and are updated
-individually. In contrast, the |CL| update process equates to installing an
-entirely new OS version with a specific set of bundles. Extremely efficient,
-updates only change files instead of entire packages.
+:command:`swupd` is an operating system software manager and update program
+that operates at a file-level. |CL| leverages :command:`swupd` to allow
```

Comment:
Ln7: Recommend omitting this line because it's partially repeated below in Versioning, Ln17. See also suggested edits below.

Reply:
Since this document is about swupd, I think it's an important opening sentence. I'm ok with the repetition in this case. I can foresee the "versioning" section standing alone and being referenced in other documents, so having the context in both places makes sense to me.